Pod/Classes/LSTracer.h

```diff
@@ -59,6 +59,9 @@ extern NSInteger const LSBackgroundTaskError;
 /// `LSTracer` instance's globally unique id ("guid"), and assigned automatically by LightStep.
 @property(nonatomic, readonly) UInt64 runtimeGuid;
 
+/// HTTP session to be used for performing requests. This enable sharing a connnection pool with your own app.
```

Comment:
non-nit: let's add a sentence like `Should be set during initialization, ideally before starting and finishing Spans.`